tests/admin/test_controller.py

```diff
@@ -179,6 +183,68 @@ def test_edit_invalid_input(self):
             eq_(400, response.status_code)
             eq_(INVALID_EDIT.uri, response.uri)
 
+    def test_update_genres(self):
+        # start with no genres
```

Comment:
it would be good to start with something and make sure the staff changes override it